integration/tests/cook/test_basic.py

```diff
@@ -917,7 +917,7 @@ def determine_user(self):
 
     def test_list_jobs_by_state(self):
         # schedule a bunch of jobs in hopes of getting jobs into different statuses
-        job_specs = [util.minimal_job(command=f"sleep {i}") for i in range(1, 20)]
+        job_specs = [util.minimal_job(command=f"sleep {i}", cpus=1) for i in range(1, 20)]
```

Comment:
Why do these need 1 cpu?

Reply:
Because the test expects some of the jobs to wait in the queue. If you make them all very small, then all of them will execute and the check for Waiting (lower down) will break.

Reply:
Discussed offline, we're going to delete this test